static/js/message_store.js

```diff
@@ -45,9 +45,6 @@ exports.get_pm_full_names = function (message) {
 
 exports.process_message_for_recent_private_messages = function (message) {
     var user_ids = people.pm_with_user_ids(message);
-    if (!user_ids) {
```

Comment:
Actually, I messed up there. I thought that was unreachable, but there are some cases I overlooked. Will put that back and add a test.